common/Root.tsx

```diff
@@ -95,7 +95,7 @@ export default class Root extends Component<Props, State> {
 
 const LegacyRoutes = withRouter(props => {
   const { history } = props;
-  const { pathname, hash } = props.location;
+  const { pathname, hash, search } = props.location;
 
   if (pathname === '/') {
     switch (hash) {
```

Comment:
Search carry-over needs to happen for `#send-transaction` and `#offline-transaction` too.